ironfish/src/rpc/response.ts

```diff
@@ -43,20 +49,25 @@ export class RpcResponse<TEnd = unknown, TStream = unknown> {
     return this as RpcResponseEnded<TEnd>
   }
 
-  async *contentStream(ignoreClose = true): AsyncGenerator<TStream, void> {
+  /*
+   * Returns a generator of stream results. If a disconnect error occurs during
+   * the streaming request it just causes the generator to end, the error is
+   * not propagated
+   */
+  async *contentStream(): AsyncGenerator<TStream, void> {
```

Comment:
Do we want to reduce this functionality for users? Before this, we let users decide before if they care if a disconnect occurs or not. Some people may want to handle the disconnect case themselves. Now it's not possible to know that a response stream was terminated because of the connection ending.

Reply:
It wasn't used anywhere which is why I removed it. + when things are in the potential refactor path I think it makes it easier if all sugar coating is kind of stripped away before so it's easier to refactor. But yea I can add it back in just so we don't forget about it